File: db_connector/models/Account.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;

const AccountSchema = new Schema({

  firstName: {
    type: String,
    required: true
  },
  lastName: {
    type: String,
    required: true
  },
  accountnumber: {
    type: String,
    required: true
  },
  prize: {
    type: String,
    required: true
  }
});

module.exports = Account = mongoose.model("accounts", AccountSchema);

File: server/src/main/java/com/qa/persistence/domain/Account.java
package com.qa.persistence.domain;

public class Account {

	private String accountnumber;

	private String firstName;
	private String lastName;

	private String prize;

	public Account() {

	}

	public Account(String accountnumber, String firstName, String lastName, String prize) {
		super();
		this.accountnumber = accountnumber;
		this.firstName = firstName;
		this.lastName = lastName;
		this.prize = prize;
	}

	public String getAccountnumber() {
		return accountnumber;
	}

	public void setAccountnumber(String accountNumber) {
		this.accountnumber = accountNumber;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getPrize() {
		return prize;
	}

	public void setPrize(String prize) {

		this.prize = prize;
	};

}
